resources/src/App.css

```diff
@@ -0,0 +1,15 @@
+body {
```

Comment:
I think the standard for css is `kebak-case` but it's ok if you want to match the component filename.

Reply:
I've used `create-react-app` to generate the project structure and the `App.css` file was included, so I would agree that this filename is acceptable because of the latter case you've mentioned.